lib/grell/capybara_driver.rb

```diff
@@ -13,7 +13,11 @@ def self.setup(options)
 
     def setup_capybara
       @poltergeist_driver = nil
-      Capybara.register_driver :poltergeist_crawler do |app|
+
+      # Capybara will not re-run the block if the driver name already exists, so the driver name
+      # will have a time integer appended to ensure uniqueness.
+      driver_name = "poltergeist_crawler_#{Time.now.to_i}".to_sym
```

Comment:
If we have a logger avaiable here (not sure if we have), I think we should log this. So we can know how things are going on long running processes